src/scenes/Partners/Edit/EditPartner.tsx

```diff
@@ -145,6 +148,7 @@ export const EditPartner = ({
         financialReportingTypes: partner.financialReportingTypes.value,
         address: partner.address.value,
         organizationName: partner.organization.value!.name.value!,
+        acronym: partner.organization.value?.acronym.value || ' ',
```

Comment:
How come you are falling back to `" "` here?

Reply:
When I passed down a null or empty string (use case: when the acronym is cleared out on the UI) the back end API would treat it as if there were no changes to update and would not clear out/update the acronym value correctly.  I didn't want to mess with the back end API, so I used this string with an empty space as a work around.